charts/gardener-extension-provider-packet/templates/configmap-logging.yaml

```diff
@@ -9,20 +9,20 @@ data:
   filter-kubernetes.conf: |
     [FILTER]
         Name                parser
-        Match               kubernetes.*cloud-controller-manager*packet-cloud-controller-manager*
+        Match               kubernetes.*cloud-controller-manager*cloud-provider-equinix-metal*
         Key_Name            log
         Parser              kubeapiserverParser
         Reserve_Data        True
 
     [FILTER]
         Name                parser
-        Match               kubernetes.*machine-controller-manager*packet-machine-controller-manager*
+        Match               kubernetes.*machine-controller-manager*equinix-metal-machine-controller-manager*
```

Comment:
I think this needs to stay as it is for now because you have not yet renamed the `machine-controller-manager` Deployment specifics but only those for `cloud-controller-manager`.

Reply:
Aha! This is why I asked you to review it. I wasn't sure if that was related to MCM or to here. Fixing right away.